autoendpoint/Cargo.toml

```diff
@@ -13,15 +13,16 @@ edition = "2018"
 # The `autoendpoint` branch merges these three PRs together.
 # The version of a2 at the time of the fork is v0.5.3.
 a2 = { git = "https://github.com/mozilla-services/a2.git", branch = "autoendpoint" }
-actix-web = "3.2"
-actix-rt = "1.1"
+# several of these libraries are pinned due to https://github.com/mozilla-services/autopush-rs/issues/249
+actix-web = "3.2"  # 3.3+ requires futures 0.3+
```

Comment:
3.3 is still on the older tokio/futures, autoendpoint's already using it according to Cargo.lock